master/internal/api_user.go

```diff
@@ -182,7 +170,20 @@ func (a *apiServer) GetUsers(
 		query.Where("u.active = ?", *req.Active)
 	}
 
-	err := query.Order(orderExpr).Scan(ctx)
+	orderBy, ok := orderByMap[req.OrderBy]
+	if !ok {
+		return nil, fmt.Errorf("unsupported order by %s", req.OrderBy)
+	}
+	sortColumn, ok := sortColMap[req.SortBy]
+	if !ok {
+		return nil, fmt.Errorf("unsupported sort by %s", req.SortBy)
+	}
+	query = query.OrderExpr("? ?", bun.Ident(sortColumn), bun.Safe(orderBy))
+	if sortColumn != "id" {
+		query = query.OrderExpr("id asc")
```

Comment:
nit: there is a lot of inconsistent usage of `query.ABC()` vs `query = query.ABC()` maybe just pick the former, it returning the query is really for the fluent interface ([explainer](https://en.wikipedia.org/wiki/Fluent_interface)).

Reply:
good point! fixing!